Makefile

```diff
@@ -0,0 +1,19 @@
+APP_NAME = varkes-mocking-suite

```

Comment:
It will be used only in prow where we will have just one single plan, so let's called it "varkes"